gc/base/GCExtensionsBase.cpp

```diff
@@ -323,6 +323,15 @@ MM_GCExtensionsBase::computeDefaultMaxHeap(MM_EnvironmentBase* env)
 	memoryMax = MM_Math::roundToFloor(heapAlignment, (uintptr_t)memoryToRequest);
 }
 
+void
+MM_GCExtensionsBase::reinitializeForRestore(MM_EnvironmentBase* env)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
+	usablePhysicalMemory = omrsysinfo_get_addressable_physical_memory();
+	gcThreadCountForced = false;
+	parSweepChunkSize = 0;
+}
```

Comment:
gcThreadCountForced is being further changed by some other code outside of MM_GCExtensionsBase, so I not sure if makes it any cleaner to reinit it here, but I'll leave it up to you (have a look yourself where else it's used)

for the other 2, yes it makes sense